### ISSUE TITLE
Linux Deploy seems to only run on single-core.

### ISSUE THREAD

**Issue by Danct12** [meefik/linuxdeploy#973]
My phone is a octa-core processor, Snapdragon 435. I run Arch Linux ARM (aarch64) and i3wm.

All working fine, but the problem is that it doesn't run on all cores, every applications only run on CPU 0, you can see this by looking in gnome-system-monitor.

Because of this, it is a pain to compile programs, is there a workaround for this? I'd happy to run on all cores.

**Comment by InvalidSudo** [comment]
This is not true on my device running Debian Stretch. I tested this on my quad-core Snapdragon 801 Android 8.1 by enabling the CPU info overlay in the OmniROM settings, closing all apps and launching Linux Deploy. When idle, both cpu0 and cpu3 are active, but when launching Chromium all cores are active. I have disabled touch boost in my device's kernel settings.

Oddly enough, in gnome-system-monitor CPU1 through CPU6 are listed (no CPU0) even though there's only 4 cores. Additionally, the monitor showed that the system was utilizing all cores, and the info matched what was shown in the overlay.

**Comment by Danct12** [comment]
Weird, I checked kernel auditor while running dd'ing zero to nothing and Core 1, 4, 5, 6 has full activities. But oddly enough that it only shows that one thread only (CPU1) on gnome-system-monitor going 100%, others are just 0%.

**Comment by Danct12** [comment]
Okay, I found out how to get them all up and working now.

Install Kernel Auditor from Google Play
Disable Core Control in Thermal and go to CPU then enable all cores.

And now I got all cores up and working!!
![image](https://user-images.githubusercontent.com/19162768/42199056-3cff5252-7eb6-11e8-9d3a-6e557b1cd036.png)

(i use dd to stress CPU on 8 threads)
